index.html

```diff
@@ -97,29 +97,17 @@ <h3>Bounds</h3>
-
-					<p>The primary entry page is the only resource in which <a href="https://w3c.github.io/wpub/indexmanifest-embedding">a manifest can be embedded</a>. To ensure discovery of the manifest, the primary entry page MUST provide a <a href="https://w3c.github.io/wpub/indexmanifest-linking">link to the manifest</a>, regardless of whether the manifest is embedded within the page or external to it.</p>
-
-					<p>The address of the primary entry page is also the <a>canonical identifier</a> for the Audiobook (i.e., it serves as the unique identifier for the Audiobook).</p>
-
-					<p>In certain cases where information has been omitted from the manifest, user agents will sometimes use the primary entry page as a fallback source of information (see <a href="https://w3c.github.io/wpub/indexlanguage-and-dir">language and base direction</a> and <a href="https://w3c.github.io/wpub/indextitle">title</a>).</p>
+					<p>It is not required that the primary entry page be included in the <a href="#audio-readingorder">default reading order</a>, as the Audiobooks profile requires that only audio resources be present. The primary entry page should instead, if present, be included as a <a href="https://w3c.github.io/wpub/index.html/#wp-resources">resource</a>.</p>
 				</section>
 
 				<section id="audio-toc">
 					<h3>Table of Contents</h3>
 
 					<p>The table of contents provides a hierarchical list of links that reflects the structural outline of the major sections of the Audiobook.</p>
 
-					<p>The table of contents is expressed via an [[html]]&#160;element (typically a <a href="https://www.w3.org/TR/html/sections.html#the-nav-element"><code>nav</code> element</a>) in one of the <a href="https://w3c.github.io/wpub/indexwp-resources">resources</a>. This element MUST be identified by the <code>role</code> attribute&#160;[[html]] value "<code>doc-toc</code>"&#160;[[dpub-aria-1.0]], and MUST be the first element in the document &#8212; in <a href="https://dom.spec.whatwg.org/#concept-tree-order">document tree order</a>&#160;[[dom]] &#8212; with that <code>role</code> value.</p>
-
-					<p>If the table of contents is not located in the <a href="https://w3c.github.io/wpub/index#wp-primary-entry-page">primary entry page</a>, the manifest SHOULD <a href="#table-of-contents">identify the resource</a> that contains the structure.</p>
+					<p>The Web Publication specification [[wpub]] defines the <a href="https://w3c.github.io/wpub/index.html#wp-table-contents">general structure</a> of the Table of Contents, defined in HTML [[html]].</p>
```

Comment:
Minor nit, but "define" is used twice in the sentence. I think you can just drop the comma and second instance so it reads  "defines the general structure ... in HTML"